scalafix-cli/src/main/scala/scalafix/cli/ScalafixOptions.scala

```diff
@@ -44,85 +50,87 @@ case class ScalafixOptions(
     @ExtraName("c")
     configStr: Option[String] = None,
     @HelpMessage(
-      """Absolute path passed to scalahost with -P:scalahost:sourceroot:<path>.
-        |        Relative filenames persisted in the Semantic DB are absolutized
-        |        by the sourceroot. Required for semantic rewrites.
-      """.stripMargin)
+      "Absolute path passed to scalahost with -P:scalahost:sourceroot:<path>. " +
+        "Relative filenames persisted in the Semantic DB are absolutized by the " +
+        "sourceroot. Required for semantic rewrites.")
     @ValueDescription("/foo/myproject")
     sourceroot: Option[String] = None,
     @HelpMessage(
-      """java.io.File.pathSeparator separated list of jar files or directories
-        |        containing classfiles and `semanticdb` files. The `semanticdb`
-        |        files are emitted by the scalahost-nsc compiler plugin and
-        |        are necessary for the semantic API to function. The
-        |        classfiles + jar files are necessary forruntime compilation
-        |        of quasiquotes when extracting symbols (that is,
-        |        `q"scala.Predef".symbol`)""".stripMargin
+      "java.io.File.pathSeparator separated list of directories containing " +
```

Comment:
Although, I have no idea what will happen if a jar is passed in. I suspect scalafix will try to fix those files too :O Do we have any tests in scalameta/scalameta for reading/writing .semanticdb files from jars?